source/code/scxsystemlib/common/scxostypeinfo.cpp

```diff
@@ -52,7 +52,7 @@ using namespace SCXCoreLib;
 
 namespace {
 
-#if defined(linux)
+#if defined(linux) && !defined(PF_DISTRO_ULINUX)
```

Comment:
Opposite.  The routine is only called when PF_DISTRO_ULINUX is Not defined and linux is defined.